assets/multiqc_config.yml

```diff
@@ -3,9 +3,9 @@ custom_logo_url: https://github.com/nf-core/sarek/
 custom_logo_title: "nf-core/sarek"
 
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/sarek" target="_blank">nf-core/sarek</a>
+  This report has been generated by the <a href="https://github.com/nf-core/sarek/3.3dev" target="_blank">nf-core/sarek</a>
   analysis pipeline. For information about how to interpret these results, please see the
-  <a href="https://nf-co.re/sarek" target="_blank">documentation</a>.
+  <a href="https://nf-co.re/sarek/3.3dev/output" target="_blank">documentation</a>.
```

Comment:
@maxulysse : Is that as it should be?

Reply:
Yeah, I'm assuming it'll be tied to the version now